nsdb-http/src/main/scala/io/radicalbit/nsdb/web/validation/Validators.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.radicalbit.nsdb.web.validation
+
+import io.radicalbit.nsdb.sql.parser.RegexNSDb
+import io.radicalbit.nsdb.web.routes.InsertBody
+
+object Validators extends RegexNSDb {
+
+  implicit val insertBodyValidator: Validator[InsertBody] = new Validator[InsertBody] {
+
+    private def metricRule(metricName: String): Boolean = !metricName.matches(metric.regex)
```

Comment:
we could consider implementing a generic method that validated a string against a regexp